src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import "../styles/variables";
+@import "../styles/blocks/card";
+@import "../styles/blocks/stars";
+@import "../styles/button";
+
+:root {
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
We don't set values here, If you want to set up a font-family for the whole page, do it in the `body` or `html` 
In `:root` we only set CSS variables (if we use them)